SU-SWS/stanford_capx
# 7.x-2.2-alpha2

# READY FOR REVIEW


# Summary
- Prep for a `7.x-2.2-alpha2` release

# Needed By (Date)
- When does this need to be merged by? 
    - December 1st? Sooner is fine too.

# Criticality
- How critical is this PR on a 1-10 scale? 5
- Would like to have a tagged release for ACSF before launching the Earth pilot

# Steps to Test

1. Check out this branch
2. Run `git diff --name-only 7.x-2.x` and verify that only `.info`, `CHANGELOG`, and `README` files have been changed
3. Run `grep -r 7.x-2.2-dev stanford_capx` and verify that none of the files have `7.x-2.2-dev` in them
4. Merge
5. Release

# Affected Projects or Products
- ACSF
- Earth Pilot

# Associated Issues and/or People
- JIRA ticket: https://stanfordits.atlassian.net/browse/SITES-260
- Anyone who should be notified? ( @kbrownell )
